### ISSUE TITLE
Installing heartbeat-6.0.0-rc1 on debian does not work

### ISSUE THREAD

**Issue by botzill** [elastic/beats#5411]
- Version: 
`6.0.0-rc1`
- Operating System: 
`Linux frontend 3.16.0-4-amd64 #1 SMP Debian 3.16.36-1+deb8u2 (2016-10-19) x86_64 GNU/Linux`
- Steps to Reproduce: 
`sudo dpkg -i heartbeat-6.0.0-rc1-amd64.deb`

After I install the package there is no files in `/etc/heartbeat`(heartbeat.yml, fields.yml) and the `heartbeat` service is not started by default.

Are the any issues with .deb package?

dpkg debug output:
```
sudo dpkg -i --debug=10 heartbeat-6.0.0-rc1-amd64.deb
D000010: ensure_pathname_nonexisting '/var/lib/dpkg/tmp.ci'
(Reading database ... 37836 files and directories currently installed.)
Preparing to unpack heartbeat-6.0.0-rc1-amd64.deb ...
Unpacking heartbeat-elastic (6.0.0-rc1) over (6.0.0-rc1) ...
D000010: tarobject ti->name='.' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/.' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/bin' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/bin' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/bin/heartbeat' mode=100775 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/bin/heartbeat' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/bin/heartbeat.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/bin/heartbeat.dpkg-tmp'
D000010: tarobject ti->name='./usr/share' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/bin' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/bin' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/bin/heartbeat' mode=100755 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/bin/heartbeat' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/bin/heartbeat-god' mode=100755 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/bin/heartbeat-god' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat-god.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat-god.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/default' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/default' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/default/dashboard' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/default/dashboard' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/default/index-pattern' mode=40775 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/default/index-pattern' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json' mode=100664 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/search' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/search' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/dashboard' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/dashboard' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/index-pattern' mode=40775 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/index-pattern' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json' mode=100664 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/NOTICE.txt' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/NOTICE.txt' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/NOTICE.txt.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/NOTICE.txt.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/LICENSE.txt' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/LICENSE.txt' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/LICENSE.txt.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/LICENSE.txt.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/README.md' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/README.md' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/README.md.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/README.md.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/heartbeat/.build_hash.txt' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/heartbeat/.build_hash.txt' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/.build_hash.txt.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/.build_hash.txt.dpkg-tmp'
D000010: tarobject ti->name='./usr/share/doc' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/doc' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/doc/heartbeat-elastic' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/usr/share/doc/heartbeat-elastic' flags=2 instead='<none>'
D000010: tarobject ti->name='./usr/share/doc/heartbeat-elastic/changelog.gz' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/doc/heartbeat-elastic/changelog.gz' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/usr/share/doc/heartbeat-elastic/changelog.gz.dpkg-new'
D000010: ensure_pathname_nonexisting '/usr/share/doc/heartbeat-elastic/changelog.gz.dpkg-tmp'
D000010: tarobject ti->name='./etc' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/etc' flags=2 instead='<none>'
D000010: tarobject ti->name='./etc/heartbeat' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/etc/heartbeat' flags=2 instead='<none>'
D000010: tarobject ti->name='./etc/heartbeat/fields.yml' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/etc/heartbeat/fields.yml' flags=7 instead='<none>'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/fields.yml.dpkg-new'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/fields.yml.dpkg-tmp'
D000010: tarobject ti->name='./etc/heartbeat/heartbeat.yml' mode=100600 owner=0:0 type=48(-) ti->linkname='' namenode='/etc/heartbeat/heartbeat.yml' flags=7 instead='<none>'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/heartbeat.yml.dpkg-new'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/heartbeat.yml.dpkg-tmp'
D000010: tarobject ti->name='./etc/heartbeat/heartbeat.reference.yml' mode=100640 owner=0:0 type=48(-) ti->linkname='' namenode='/etc/heartbeat/heartbeat.reference.yml' flags=7 instead='<none>'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/heartbeat.reference.yml.dpkg-new'
D000010: ensure_pathname_nonexisting '/etc/heartbeat/heartbeat.reference.yml.dpkg-tmp'
D000010: tarobject ti->name='./etc/init.d' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/etc/init.d' flags=2 instead='<none>'
D000010: tarobject ti->name='./etc/init.d/heartbeat-elastic' mode=100755 owner=0:0 type=48(-) ti->linkname='' namenode='/etc/init.d/heartbeat-elastic' flags=7 instead='<none>'
D000010: ensure_pathname_nonexisting '/etc/init.d/heartbeat-elastic.dpkg-new'
D000010: ensure_pathname_nonexisting '/etc/init.d/heartbeat-elastic.dpkg-tmp'
D000010: tarobject ti->name='./lib' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/lib' flags=2 instead='<none>'
D000010: tarobject ti->name='./lib/systemd' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/lib/systemd' flags=2 instead='<none>'
D000010: tarobject ti->name='./lib/systemd/system' mode=40755 owner=0:0 type=53(d) ti->linkname='' namenode='/lib/systemd/system' flags=2 instead='<none>'
D000010: tarobject ti->name='./lib/systemd/system/heartbeat-elastic.service' mode=100664 owner=0:0 type=48(-) ti->linkname='' namenode='/lib/systemd/system/heartbeat-elastic.service' flags=2 instead='<none>'
D000010: ensure_pathname_nonexisting '/lib/systemd/system/heartbeat-elastic.service.dpkg-new'
D000010: ensure_pathname_nonexisting '/lib/systemd/system/heartbeat-elastic.service.dpkg-tmp'
D000010: deferred extract of '/.'
D000010: deferred extract of '/usr'
D000010: deferred extract of '/usr/bin'
D000010: deferred extract of '/usr/bin/heartbeat'
D000010: deferred extract of '/usr/share'
D000010: deferred extract of '/usr/share/heartbeat'
D000010: deferred extract of '/usr/share/heartbeat/bin'
D000010: deferred extract of '/usr/share/heartbeat/bin/heartbeat'
D000010: deferred extract of '/usr/share/heartbeat/bin/heartbeat-god'
D000010: deferred extract of '/usr/share/heartbeat/kibana'
D000010: deferred extract of '/usr/share/heartbeat/kibana/default'
D000010: deferred extract of '/usr/share/heartbeat/kibana/default/dashboard'
D000010: deferred extract of '/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/default/index-pattern'
D000010: deferred extract of '/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/search'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/dashboard'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/index-pattern'
D000010: deferred extract of '/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json'
D000010: deferred extract of '/usr/share/heartbeat/NOTICE.txt'
D000010: deferred extract of '/usr/share/heartbeat/LICENSE.txt'
D000010: deferred extract of '/usr/share/heartbeat/README.md'
D000010: deferred extract of '/usr/share/heartbeat/.build_hash.txt'
D000010: deferred extract of '/usr/share/doc'
D000010: deferred extract of '/usr/share/doc/heartbeat-elastic'
D000010: deferred extract of '/usr/share/doc/heartbeat-elastic/changelog.gz'
D000010: deferred extract of '/etc'
D000010: deferred extract of '/etc/heartbeat'
D000010: deferred extract of '/etc/heartbeat/fields.yml'
D000010: deferred extract of '/etc/heartbeat/heartbeat.yml'
D000010: deferred extract of '/etc/heartbeat/heartbeat.reference.yml'
D000010: deferred extract of '/etc/init.d'
D000010: deferred extract of '/etc/init.d/heartbeat-elastic'
D000010: deferred extract of '/lib'
D000010: deferred extract of '/lib/systemd'
D000010: deferred extract of '/lib/systemd/system'
D000010: deferred extract of '/lib/systemd/system/heartbeat-elastic.service'
D000010: process_archive looking for overwriting '/usr/bin/heartbeat'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/bin/heartbeat'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/bin/heartbeat-god'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/NOTICE.txt'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/LICENSE.txt'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/README.md'
D000010: process_archive looking for overwriting '/usr/share/heartbeat/.build_hash.txt'
D000010: process_archive looking for overwriting '/usr/share/doc/heartbeat-elastic/changelog.gz'
D000010: process_archive looking for overwriting '/etc/heartbeat/fields.yml'
D000010: process_archive looking for overwriting '/etc/heartbeat/heartbeat.yml'
D000010: process_archive looking for overwriting '/etc/heartbeat/heartbeat.reference.yml'
D000010: process_archive looking for overwriting '/etc/init.d/heartbeat-elastic'
D000010: process_archive looking for overwriting '/lib/systemd/system/heartbeat-elastic.service'
D000010: ensure_pathname_nonexisting '/usr.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/bin.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/bin/heartbeat.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/bin/heartbeat-god.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/dashboard.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/dashboard/Heartbeat-http-monitor.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/index-pattern.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/default/index-pattern/heartbeat.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/search.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/search/02014c80-29d2-11e7-a68f-bfaa2341cc52.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/dashboard.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/dashboard/f3e771c0-eb19-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/1738dbc0-eb1d-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/091c3a90-eb1e-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/c65ef340-eb19-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/920e8140-eb1a-11e6-be20-559646f8b9ba.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/visualization/0f4c0560-eb20-11e6-9f11-159ff202874a.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/index-pattern.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/kibana/5.x/index-pattern/heartbeat.json.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/NOTICE.txt.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/LICENSE.txt.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/README.md.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/heartbeat/.build_hash.txt.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/doc.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/doc/heartbeat-elastic.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/usr/share/doc/heartbeat-elastic/changelog.gz.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/etc.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/etc/heartbeat.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/etc/init.d.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/lib.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/lib/systemd.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/lib/systemd/system.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/lib/systemd/system/heartbeat-elastic.service.dpkg-tmp'
D000010: ensure_pathname_nonexisting '/var/lib/dpkg/tmp.ci'
D000010: ensure_pathname_nonexisting running rm -rf '/var/lib/dpkg/tmp.ci'
D000010: ensure_pathname_nonexisting '/var/lib/dpkg/reassemble.deb'
Setting up heartbeat-elastic (6.0.0-rc1) ...
Processing triggers for systemd (215-17+deb8u5) ...
```

**Comment by botzill** [comment]
Sorry about the issue above, now seems to work OK, closing  it.

**Comment by thewasta** [comment]
> Sorry about the issue above, now seems to work OK, closing it.

How? How did you fixed it? I'm using 6.6.1 and I have the same problem, it doesn't create any files.
Thanks
